src/shell/layout/tiling/grabs/resize.rs

```diff
@@ -94,6 +94,7 @@ impl PointerTarget<State> for ResizeForkTarget {
                         location.as_global(),
                         node,
                         left_up_idx,
+                        None,
```

Comment:
Preferably these would also gain the option to do more complex grabs at corners, but I guess we can leave this for now.